aspnetcore/signalr/background-services.md

```diff
@@ -68,11 +53,29 @@ For more information on strongly typed Hubs, see [Use hubs in SignalR for ASP.NE
 > [!NOTE]
 > This functionality isn't limited to the [Hub\<T>](xref:Microsoft.AspNetCore.SignalR.Hub`1) class. Any class that inherits from [Hub](xref:Microsoft.AspNetCore.SignalR.Hub), such as [DynamicHub](xref:Microsoft.AspNetCore.SignalR.DynamicHub), will also work.
```

Comment:
```suggestion
> This functionality isn't limited to the [Hub\<T>](xref:Microsoft.AspNetCore.SignalR.Hub`1) class. Any class that inherits from [Hub](xref:Microsoft.AspNetCore.SignalR.Hub), such as [DynamicHub](xref:Microsoft.AspNetCore.SignalR.DynamicHub) works.
```

Reply:
After looking at this a bit more I think it reads better if the comma and the future tense are kept, but I agree that the "also" is not needed. Happy to defer on this if you disagree, but I'm going to lay the 3 out side by side for easy comparison as it's tricky to really feel out the sentence when there's markdown everywhere.

>Any class that inherits from Hub, such as DynamicHub, will also work.

>Any class that inherits from Hub, such as DynamicHub works.

>Any class that inherits from Hub, such as DynamicHub, will work.

Reply:
@UncleDave  the MS style guide specifically recommends avoiding future tense. To the untrained ear, FT often sounds better. It doesn't machine translate and it's actually no clearer.

Reply:
Fair enough, how do we feel about just restoring the bracketing comma then?

>Any class that inherits from Hub, such as DynamicHub, works.

Reply:
That's fine